mk/utils/glide.mk

```diff
@@ -0,0 +1,28 @@
+# Cross builder helper
+define glide-install
+	glide install -s --update-vendored
+endef
+
+define glide-update
+	glide update -s --update-vendored
```

Comment:
On our project we commit the libraries in the vendor folder, so that we still have reproducible builds in case the project our original repo poofs away.   How will glide support that in the future?  Will it simply move the .git folders to a different folder, out of the way of the main repo?  Only reason we need the options are to avoid having custom script or step that would clean up all the .git folders from vendor.